azure-iot-device/azure/iot/device/iothub/pipeline/config.py

```diff
@@ -25,3 +25,8 @@ def __init__(self, product_info="", **kwargs):
         """
         super(IoTHubPipelineConfig, self).__init__(**kwargs)
         self.product_info = product_info
+
+        # Now, the parameters below are not exposed to the user via kwargs. They need to be set by manipulating the IoTHubPipelineConfig object.
+        # They are not in the BasePipelineConfig because these do not apply to the provisioning client.
+        self.blob_upload = False
+        self.method_invoke = False
```

Comment:
I wonder if these might be better in some sort of structure. Again, not super important now, but something to consider.